phonenumber_field/widgets.py

```diff
@@ -28,8 +30,16 @@ def __init__(self, initial=None):
                         choices.append((prefix, "{} {}".format(country_name, prefix)))
         super().__init__(choices=sorted(choices, key=lambda item: item[1]))
 
-    def render(self, name, value, *args, **kwargs):
-        return super().render(name, value or self.initial, *args, **kwargs)
+    def create_option(self, name, value, label, selected, index, subindex=None, attrs=None):
+        if not value:
+            selected = False
+        if not self.has_selected_value and self.initial is not None and value == self.initial:
+            selected = True
+        return super(PhonePrefixSelect, self).create_option(name, value, label, selected, index, subindex, attrs)
```

Comment:
This project is Python 3 only, so you can simplify this to:

```suggestion
        return super().create_option(name, value, label, selected, index, subindex, attrs)
```